Sources/MiniPlayer/AudioMiniPlayer.swift

```diff
@@ -0,0 +1,168 @@
+/*****************************************************************************
+ * AudioMiniPlayer.swift
+ * VLC for iOS
+ *****************************************************************************
+ * Copyright (c) 2019 VideoLAN. All rights reserved.
+ * $Id$
+ *
+ * Authors: Soomin Lee <bubu # mikan.io>
+ *
+ * Refer to the COPYING file of the official project for license.
+ *****************************************************************************/
+
+@objc(VLCAudioMiniPlayer)
+class AudioMiniPlayer: UIView, MiniPlayer {
+    var visible: Bool = false
+    var contentHeight: Float {
+        return 72.0
+    }
+
+    @IBOutlet private weak var audioMiniPlayer: UIView!
+    @IBOutlet private weak var artworkImageView: UIImageView!
+    @IBOutlet private weak var titleLabel: UILabel!
+    @IBOutlet private weak var artistLabel: UILabel!
+    @IBOutlet private weak var progressBarView: UIProgressView!
+    @IBOutlet private weak var playPauseButton: UIButton!
+    @IBOutlet private weak var previousButton: UIButton!
+    @IBOutlet private weak var nextButton: UIButton!
+
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        initView()
+        setupConstraint()
+    }
+
+    required init?(coder aDecoder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    func updatePlayPauseButton() {
+        let isPlaying = VLCPlaybackController.sharedInstance().isPlaying
+        playPauseButton.isSelected = !isPlaying
```

Comment:
same as above, rather set a new image instead of toggling the selected state and maybe even the correct accessibilityidentifier?

Reply:
To have consistency with our accessibility labels, I thing I will use `PLAY_PAUSE_BUTTON` instead of `PLAY_BUTTON`.